deploy/clowdapp.yaml

```diff
@@ -166,6 +178,12 @@ objects:
                   value: "true"
                 - name: INSIGHTS_RESULTS_AGGREGATOR__LOGGING__LOGGING_TO_CLOUD_WATCH_ENABLED
                   value: ${CLOUDWATCH_ENABLED}
+                - name: INSIGHTS_REDIS_URL
+                  value: "ccx-redis:6379"
```

Comment:
just FYI this might change depending on what we call the service

Reply:
na dw, let's call it `ccx-redis` :)